examples/custom-icon.js

```diff
@@ -0,0 +1,27 @@
+goog.require('ol.Map');
+goog.require('ol.View');
+goog.require('ol.layer.Tile');
+goog.require('ol.source.OSM');
+
+var logoElement = document.createElement('a');
+logoElement.href = 'http://www.osgeo.org/';
+logoElement.target = '_blank';
+
+var logoImage = new Image();
```

Comment:
Without having tried, but maybe changing this line to `var logoImage = document.createElement('img');` fixes the problem you described.